### PR TITLE
fix: various fixes

### LINKED ISSUE #1240: [Bug]: Widgets only refresh when you close and reopen the application
### Describe the bug

We left the Widgets open on three phones and they only refresh when the application is closed and reopened.

### Reproduce

1. Go to 'widget'
2. Select the widget
3. Wait few min

### Screenshots / Recording

_No response_

### Operating system

Graphene OS and iOS

### Bitkit version

v46

### Log output

_No response_

### LINKED ISSUE #1244: [Feature]: Support Umbrel Electrum connection QR
### Describe the problem

When scanning a QR from the electrum server settings screen Bitkit should support the following QR in the format of `<host>:<port>:<t|s>`

Screenshot 2023-09-11 at 11 50 34

### Describe the solution

_No response_

### Additional context

_No response_

### LINKED ISSUE #1249: [Bug]: Part of widget preview screen hidden behind another layer
### Describe the bug

I noticed that when we add the widget, or when we click to see its settings, a screen appears with part of the widget appearing behind another layer of the app. Is it possible to make everything fit there without the user having to scroll down? Otherwise, I think it would be better if the user could scroll down to see the preview of the widget.

### Reproduce

1. Add a Lugano Widget or paste this url: `slashfeed:s8y5x6hjgqx54yk7ycgyd5q16z5hthnxq9msn6w1wmnmphwtgz3y/PlanB forum?relay=https://dht-relay.synonym.to/staging/web-relay`

### Screenshots / Recording

simulator_screenshot_B7C3456B-2755-4B2C-B91D-6B171664D7CA

### Operating system

IOS 16.4

### Bitkit version

_No response_

### Log output

_No response_

### PR DESCRIPTION
**Pull request by pwltr**
### Description

- feat(settings): #1244 support Umbrel connection QR
- fix(widgets): switch to polling for price widget
- fix(widgets): remove hardcoded lugano widget icon
- fix(profile): remove duplicate tooltip when copying slashtag
- fix(send): show transaction fee as fiat on Review screen
- fix: hide error when clearing already empty async storage
- fix(wallet): decrease BalanceHeader pressable area
- refactor: navigator animation options
- test: fix flaky e2e tests

### Linked Issues/Tasks

Closes https://github.com/synonymdev/bitkit/issues/1244 https://github.com/synonymdev/bitkit/issues/1249 https://github.com/synonymdev/bitkit/issues/1240

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactoring (improving code without creating new functionality)

### Tests

- [x] Detox test
- [ ] Unit test
- [ ] No test

### CODE REVIEW

File: e2e/slashtags.e2e.js [hunk 1]
```diff
@@ -127,8 +129,7 @@ d('Profile and Contacts', () => {
 
 			// self
 			await element(by.id('AddContact')).tap();
-			await element(by.id('ContactURLInput')).typeText(slashtagsUrl + '\n');
-			// await element(by.id('ContactURLInput')).tapReturnKey();
+			await element(by.id('ContactURLInput')).replaceText(slashtagsUrl);
```
Comment (limpbrains) [inline e2e/slashtags.e2e.js]: It doesn't work **without** hardware keyboard attached

File: src/screens/Wallets/Send/ReviewAndSend.tsx [hunk 2]
```diff
@@ -627,8 +629,11 @@ const ReviewAndSend = ({
 								value={
 									<View style={styles.fee}>
 										{feeIcon}
-										<Text02M>{t(`fee:${selectedFeeId}.title`)} </Text02M>
-										<Money sats={feeSats} size="text02m" />
+										<Text02M>
+											{t(`fee:${selectedFeeId}.title`)} (
+											{fiatTransactionFee.fiatSymbol}
+											{fiatTransactionFee.fiatFormatted})
+										</Text02M>
```
Comment (limpbrains) [inline src/screens/Wallets/Send/ReviewAndSend.tsx:636]: If you need to force Fiat here, you can do:
<Money unit="fiat" ...

File: e2e/lnurl.e2e.js [hunk 3]
```diff
@@ -146,7 +148,7 @@ d('LNURL', () => {
 			if (peers.some((p) => p.pubKey === ldkNodeID)) {
 				break;
 			}
-			if (n++ === 19) {
+			if (n++ === 29) {
```
Comment (limpbrains) [inline e2e/lnurl.e2e.js]: Does this relly help or it is just a guess ?

**Review by limpbrains — CHANGES_REQUESTED** [review]

File: e2e/slashtags.e2e.js (hunk 1 above)
Comment (limpbrains) [reply e2e/slashtags.e2e.js:132]: * without

File: e2e/lnurl.e2e.js (hunk 3 above)
Comment (pwltr) [reply e2e/lnurl.e2e.js]: No it doesn't fix it, reverted

File: src/screens/Wallets/Send/ReviewAndSend.tsx (hunk 2 above)
Comment (pwltr) [reply src/screens/Wallets/Send/ReviewAndSend.tsx:636]: It doesn't look the same

**Review by limpbrains — APPROVED** [review]

**Pull request merged**